pom.xml

```diff
@@ -54,7 +54,8 @@
     <ver.junit>4.12</ver.junit>  
 
     <ver.slf4j>1.7.30</ver.slf4j>
-    <ver.log4j2>2.13.1</ver.log4j2>
+    <ver.log4j2>2.13.3</ver.log4j2>
+    <ver.shade-plugin>3.2.3</ver.shade-plugin>
```

Comment:
In trying to get the 3rd party shade transformer to work, upgrading the shade plugin and the log4j2 version were necessary so evenif not required after all, the x.x.1 updates are left in.